fabfile/adage_server.py

```diff
@@ -210,6 +210,7 @@ def init_instance():
 def build_interface():
     """ have grunt perform a deployment build for us """
     with cd(CONFIG['interface_dir']), \
+            shell_env(PHANTOMJS_BIN='/usr/local/bin/phantomjs'), \
```

Comment:
Okay, here is the story on this:

I just got through a test deployment on AWS using the copy of `phantomjs` installed in `node_modules` as @dongbohu mentions above. It did not work.

It turns out that the binary that gets installed with the copy of `karma-phantomjs-launcher` (which is actually installed as `node_modules/phantomjs-prebuilt/bin/phantomjs`) doesn't work (for some unknown reason) on the AWS deployment server. That must have been the root cause of the trouble I was having with the AWS deployment and the manually-installed copy of `phantomjs` worked around it.

I completely agree with @dongbohu that we should be using what's packaged with `karma-phantomjs-launcher` and, in an ideal world, I would debug why it's not working and apply that fix. Given that we are on a deadline, however, I think it's a better use of our time to stick with this workaround for now and have another look at fixing this after our deadline.

We are also planning to transition to a Docker deployment, and that approach may make this whole problem moot anyway. In fact, the copy of `phantomjs` in `node_modules` on my VMware development instance works fine, unlike whatever is happening on AWS. This is probably some obscure node packaging issue. I'll file this as an issue to check out along with converting to Docker. Are you okay with that?